components/transparent.py

```diff
@@ -0,0 +1,26 @@
+from components import Component
+from canvas import Canvas, Colour
+from data import Data
+
+
+class Transparent(Component):
+    """A transparent rectangle behind the data overlay for data values to stand out
+    from a white background (white racetrack, lines, etc).
+
+    Personal note: For overlay_new, there should be two rectangles on each
+    bottom corner behind the data overlay.
+    """
+
+    COLOUR = Colour.transparentBlack
+
+    def __init__(self, top_left_coords: tuple, bottom_right_coords: tuple):
```

Comment:
The `tuple` type could contain anything inside it, so our IDEs won't know to expect integers when giving us type hints. One option would be to use `Tuple[int, int]` (importing `Tuple` from `typing`), but `canvas.py` actually saves this type to a variable, so that it's both named and reusable. Consider
```python
from canvas import Coord

def __init__(self, top_left_coords: Coord ...
```